teamwork/templates/courses/view_course.html

```diff
@@ -427,18 +427,44 @@ <h3>Updates</h3>
                   <!-- USERS LIST -->
                   <div class="box box-info">
                     <div class="box-header with-border">
-                      <h3 class="box-title">Instructors</h3>
+                      <h3 class="box-title">Professor</h3>
```

Comment:
Same here, needs to stay as Instructor(s), not all of our Instructors are Professors.

Reply:
Tomato Tomato, but I switched it back